src/main/java/org/zalando/nakadi/validation/EventTypeOptionsValidator.java

```diff
@@ -28,11 +27,11 @@ public void validate(final Object target, final Errors errors) {
     }
 
     private void checkRetentionTime(final Errors errors, final EventTypeOptions options) {
-        if (Objects.nonNull(options) && Objects.nonNull(options.getRetentionTime())) {
-            final Long retentionTime = options.getRetentionTime();
-            if (retentionTime > maxTopicRetentionMs) {
+        final Long retentionTime = options.getRetentionTime();
+        if (retentionTime != null) {
+            if (retentionTime.longValue() > maxTopicRetentionMs) {
```

Comment:
Why you have this `longValue()`? Were there any problems with automatic unboxing?